tests/HodaTime/Util.hs

```diff
@@ -75,6 +81,22 @@ instance Arbitrary RandomStandardDate where
     d <- choose (1,28)
     return $ RandomStandardDate y m d
 
+instance Arbitrary Instant where
+  arbitrary = fromSecondsSinceUnixEpoch <$> arbitrary
+
+instance Arbitrary Offset where
+  arbitrary = do
+    s <- choose (minOffsetSeconds, maxOffsetSeconds) :: Gen Int
+    return $ fromSeconds s
+
+newtype ValidTimeZoneName = ValidTimeZoneName String deriving (Show)
+
+instance Arbitrary ValidTimeZoneName where
+  arbitrary = ValidTimeZoneName <$> elements [
+      "UTC"
+    , "Europe/Stockholm"
+    ]
```

Comment:
@jason-johnson I would like this to use the `availableZones` method from `Data.HodaTime.TimeZone`, but I haven't been able to figure out how to transform `IO [String]` to `Gen String`, so I hard-coded a couple just to get the rest of the test working. Any ideas?

Reply:
Not off the top of my head but I have the impression that I'm doing something similar with some of the tests in there already.